hm-basic-webapp/templates/hpa.yaml

```diff
@@ -1,26 +1,21 @@
 {{- if .Values.autoscaling.enabled }}
-apiVersion: autoscaling/v2beta1
+apiVersion: autoscaling/v2
 kind: HorizontalPodAutoscaler
 metadata:
-  name: {{ include "hm-basic-webapp.prefix" . }}
+  name: {{ include "hm-basic-webapp.fullname" . }}
+  labels:
+    {{- include "hm-basic-webapp.labels" . | nindent 4 }}
```

Comment:
Is this related to HPA change? if it's not needed for HPA change I'd suggest to make the change in a separate PR to discuss the naming